frontend/plugins/pos_ui/src/modules/create-pos/components/permission/permission.tsx

```
@@ -6,6 +6,7 @@ import { zodResolver } from '@hookform/resolvers/zod';
 import { PermissionFormValues, permissionSchema } from '../formSchema';
 import { useEffect, useState, useImperativeHandle, forwardRef } from 'react';
 import { IPosDetail } from '~/modules/pos-detail/types/IPos';
+import { IPosDetail } from '~/modules/pos-detail/types/IPos';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'IPosDetail'. Consider to delete it or rename it.

'IPosDetail' is defined here:


(lint/suspicious/noRedeclare)